src/phases.ts

```diff
@@ -4734,10 +4734,16 @@ export class PokemonHealPhase extends CommonAnimPhase {
     const fullHp = pokemon.getHpRatio() >= 1;
 
     const hasMessage = !!this.message;
+
+    const healBlock = pokemon.getTag(BattlerTagType.HEAL_BLOCK);
     const healOrDamage = (!fullHp || this.hpHealed < 0);
     let lastStatusEffect = StatusEffect.NONE;
 
-    if (healOrDamage) {
+    if (healBlock) {
```

Comment:
```suggestion
    if (!!healBlock && healBlock instanceof HealBlockTag) {
```
Should fix the gh pages issue